Content.Shared/CCVar/CCVars.Admin.cs

```diff
@@ -89,6 +89,12 @@ public sealed partial class CCVars
     public static readonly CVarDef<bool> ServerBanErasePlayer =
         CVarDef.Create("admin.server_ban_erase_player", false, CVar.ARCHIVE | CVar.SERVER | CVar.REPLICATED);
 
+    /// <summary>
+    ///     If true, will reset the last time the player has read the rules. This will mean on their next login they will be shown the rules again.
+    /// </summary>
+    public static readonly CVarDef<bool> ServerBanResetLastReadRules =
+        CVarDef.Create("admin.server_ban_reset_last_read_rules", true, CVar.ARCHIVE | CVar.SERVER | CVar.REPLICATED);
```

Comment:
```suggestion
        CVarDef.Create("admin.server_ban_reset_last_read_rules", true, CVar.ARCHIVE | CVar.SERVER | CVar.REPLICATED);
```

Doesn't need to be replicated